owid/catalog/meta.py

```diff
@@ -19,7 +19,7 @@ def pruned_json(cls: T) -> T:
     orig = cls.to_dict  # type: ignore
 
     # only keep non-null public variables
-    cls.to_dict = lambda self: {  # type: ignore
```

Comment:
What's the purpose of this change?

Reply:
Honestly, I don't remember. It fixed something when I was profiling (but eventually used a different approach). The problem is that `orig` is not assigned when you're creating the lambda (see [this example](https://stackoverflow.com/a/21054087/1275818)) and `lambda self, orig=orig: ...` guarantees it does.

Anyway, I can't replicate it now (could have been a mirage as well), so I'm putting the original back.